internal/test/envtest/environment.go

```diff
@@ -103,11 +103,15 @@ type RunInput struct {
 //	because our tests require access to the *Environment. We use this field to make the created Environment available
 //	to the consumer.
 //
```

Comment:
This will be merged via https://github.com/kubernetes-sigs/cluster-api/pull/8042

/hold
until https://github.com/kubernetes-sigs/cluster-api/pull/8042 is merged. Otherwise the PR is ready for review

Reply:
Removed from this PR

/hold cancel